compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -135,6 +135,8 @@
     public final BTypedescType typeDesc = new BTypedescType(this.anyType, null);
     public final BType readonlyType = new BReadonlyType(TypeTags.READONLY, null);
     public final BType anydataOrReadonly = BUnionType.create(null, anydataType, readonlyType);
+    public final BType anyAndReadonly = new BAnyType(TypeTags.ANY, null, new Name("anyAndReadonly"),
```

Comment:
why we need any and readonly, it is same as any|error isn't it ? @KavinduZoysa @MaryamZi